examples/js/loaders/GLTFLoader.js

```diff
@@ -533,6 +538,49 @@ THREE.GLTFLoader = ( function () {
 
 	};
 
+	/**
+	 * Texture Transform Extension
+	 *
+	 * Specification:
+	 */
+	function GLTFTextureTransformExtension( json ) {
+
+		this.name = EXTENSIONS.KHR_TEXTURE_TRANSFORM;
+
+	}
+
+	GLTFTextureTransformExtension.prototype.extendTexture = function ( texture, mapDef ) {
+
+		var transform = mapDef.extensions !== undefined ? mapDef.extensions[ this.name ] : undefined;
+
+		if ( transform === undefined ) return texture;
+
+		texture = texture.clone();
+
+		if ( transform.offset !== undefined ) {
+
+			texture.offset.fromArray( transform.offset );
+
+		}
+
+		if ( transform.rotation !== undefined ) {
+
+			texture.rotation = transform.rotation;
+
+		}
+
+		if ( transform.scale !== undefined ) {
+
+			texture.repeat.fromArray( transform.scale );
+
+		}
+
+		texture.needsUpdate = true;
+
+		return texture;
```

Comment:
We don't handle `texCoord`?

https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_transform#gltf-schema-updates

Reply:
Yeah, I forgot that could be overridden. But I'm not sure how we _should_ handle it at this stage... all we can really do is print a warning, except in edge cases like where uv2 is unused and uv3 is the occlusion map (and we can just swap attributes around). 

I think we should just log a warning for now, and if #14572 is merged (not ready yet) we can actually handle this correctly when `useNodes=true`.

Reply:
Agreed with warning for now.

Reply:
Added the warning. 👍